templates/etc.httpd.conf.httpd.conf.j2

```diff
@@ -5,6 +5,7 @@ ServerRoot              "{{ apache_httpd_conf_base }}"
 PidFile                 run/httpd.pid
 
 Listen                  {{ apache_server_listen_port }}
+Listen                  {{ apache_server_status_listen_port }}
```

Comment:
You can't do this. If you have overlapping Listen directives, Apache will fail to start.